src/nestor/models/contacts_book.py

```diff
@@ -54,14 +70,17 @@ def value(self, value: str):
             raise BirthdayValueError("Invalid date format. Use DD.MM.YYYY") from exc
     
 class Contact:
+    NOT_SPECIFIED_FIELD_VALUE = 'not specified'
```

Comment:
better to create constants.py and move it there, because it could be used in other classes